### PR TITLE
feat: increase granularity of timestream log

### PR DESCRIPTION
**Pull request by rem1niscence**
hard PR, check carefully

### CODE REVIEW

File: src/services/metrics-recorder.ts [hunk 1]
```diff
@@ -213,7 +213,7 @@ export class MetricsRecorder {
       const timeStreamMeasure = {
         Dimensions: timeStreamDimensions,
         MeasureName: 'elapsedTime',
-        MeasureValue: `${parseFloat(elapsedTime.toFixed(4))}`,
+        MeasureValue: `${parseFloat(elapsedTime.toString())}`,
```
Comment (nymd) [inline src/services/metrics-recorder.ts:216]: wrong time :D  granularity of the timestamp, not this one

**Review by nymd — CHANGES_REQUESTED** [review]

File: src/services/metrics-recorder.ts (hunk 1 above)
Comment (nymd) [reply src/services/metrics-recorder.ts:216]: image

Comment (nymd) [reply src/services/metrics-recorder.ts:216]: That time on the right col

Comment (rem1niscence) [reply src/services/metrics-recorder.ts:216]: on that case both can be changed, according to the error:

```
  {
    Reason: 'A record already exists with the same time, dimensions, measure name, and record version. A higher record version must be specified in order to update the measure value. Specifying record version is supported by the latest SDKs.',
    RecordIndex: 0
  }
```
dimension values are also taken into account, but that's good so even more granularity can be achieved, will do the change

Comment (rem1niscence) [reply src/services/metrics-recorder.ts:216]: After a bit of research came to the conclusion that nodejs doesn't natively supports microseconds and after trying a few recommendations they didn't work reliably, recommend at the moment merge with only this change to provide slightly better granularity

**Review by nymd — APPROVED** [review]

**Pull request merged**